src/main.rs

```diff
@@ -159,13 +158,13 @@ fn team(t: String, subject: String) -> Template {
     Template::render(page, &context)
 }
 
-fn get_type_from_string(s: &str) -> Result<GroupType, Error> {
+fn get_type_from_string(s: &str) -> Result<GroupType, ()> {
     match s {
         "wgs" => Ok(GroupType::WorkingGroup),
         "teams" => Ok(GroupType::Team),
         "peers" => Ok(GroupType::Peer),
         "shepards" => Ok(GroupType::Shepard),
-        _ => Err(Error::Internal),
+        _ => Err(()),
```

Comment:
is there a reason we don't want to return an error type here anymore?

Reply:
The `Error` enum was removed from Rocket, and given that we don't do anything with the error and it doesn't have any data, I thought using `()` was better than creating our own error type